### DISCUSSION TITLE
Un-supported Token?

### DISCUSSION THREAD

**Discussion by canuckasaurus** [phantom/docs#324] [Help & Support] [answered]
### Summary

Created a token 4 days ago with Token 2022 program but doesn't appear in wallet at all.

https://solscan.io/token/AAjaHzRkMmbHJ9RSJvMPREgj3CAsyxyrCUXKVWyVxCSu

### Steps to Reproduce

Tokens not appearing in wallets.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@canuckasaurus 

We have a reference implementation using the metadata extensions here https://solana.fm/address/EGKdqrXxFeTpRTskrH81xmefuTAU9MJGCDLoNss28a2/metadata?cluster=devnet-solana

I would try removing the extraneous fields in your metadata (not the extension). That's the big difference I see. If this doesn't resolve the issue, please let me know.

**Comment by canuckasaurus** [comment]
There isn't... that's just the metaplex version... if you look at the json -- we actually don't have an external url that's the only difference.

https://beermonkey.space/metadata.json

**Reply by adamdelphantom** [reply, marked as answer]
I think the issue is that it appears this mint was created as an NFT. You have 0 decimals and  fields such as collection and creators that are not applicable for fungibles.

![Screenshot 2024-12-04 at 1 03 27 PM](https://github.com/user-attachments/assets/425f239f-70ee-473d-b0a5-1b338b3e95b5)

**Comment by canuckasaurus** [comment]
I know you said not to worry about the extension... but could it be that?
